exercise/reduce.js

```diff
@@ -0,0 +1,54 @@
+/*
+## Reduce
```

Comment:
Confusing name, why is it called reduce ?
Also, what has this exercise to do with callbacks ?

Reply:
is for the students have to use reduce, i can think of some other way for them to use it